content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,190 @@
+
+Read on to discover more details about the improvement with Sourcegraph 4.0.  
+
+### 🌅 Code intelligence: more useful, more times daily for every dev
+
+
+<Badge text="Code Search" color="cerise" size="small" onClick={() => window.open("/code-search", "_self")} />
+
+
+#### A faster, simpler search experience 
+We streamlined and simplified the search user interface to make the platform more intuitive. The search homepage now includes contextual tips and recommendations using real data from your instance, including repository names, file names, and authors. This information makes it easier than ever run a successful search query, faster.
+
+Search results are now front and center with the improved interface. The search sidebar has moved to the right of the screen and several non-essential elements have been removed to ensure the information you’re looking for is easy to access. Filters make it easier and faster for you to extract the answer you need, and the side bar can be collapsed altogether to reduce noise. 
+
+Performance across the platform saw a boost and search results are now returned faster than before by using the latest in React, lazy loading, and priority rendering. 
+
+![UI Improvements](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search%20UI%20Improvements.png "UI Improvements")
+
+<Badge text="Code Insights" color="green" size="small" onClick={() => window.open("/code-insights", "_self")} />
+
+#### Understand usage and search structure with high-level aggregations of search results
+It’s now easier for you to get high-level knowledge from the codebase and answer questions with aggregations of search results powered by Code Insights. In the chart that appears on the search results page, you can choose to group your search results by location (repo or file), author, or arbitrary capture group pattern.
+
+The new aggregations answer:
+- How many different versions of a library or package are present in your code, and which is most common? 
+- Which files have the majority of these search results?
+- Which repositories is this library used in most?
+- Which library components are used most? By who?
+- What are all the possible arguments we pass to this function? 
+
+Here are some [example search aggregations to get started with](https://docs.sourcegraph.com/code_insights/references/search_aggregations_use_cases)
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Search_Aggregations_4.0',
+    mp4: 'blog/release-post/4.0/Search_Aggregations_4.0'
+  }}
+  loop={true}
+  title="High-level aggregations"
+  caption="High-level aggregations of search results"
+/>
+
+
+[Docs](https://docs.sourcegraph.com/code_insights/explanations/search_results_aggregations)
+
+#### Configure precise code navigation for 9 languages in a matter of minutes with auto-indexing
```

Comment:
@iskyOS - can you please review auto-indexing. Do you think it's worth mentioning code graph data?

Reply:
Reviewed and commented below! 

I think we can mention code graph data in the spot below, but not everyone in the audience will recognize it. We could link to the docs there too: https://docs.sourcegraph.com/code_navigation/explanations/uploads

Reply:
The url has been added